lib/buttons-and-links/__tests__/KButton.spec.js

```diff
@@ -1,109 +1,75 @@
+      const wrapper = shallowMount(KButton, {
+        propsData: {
+          text: 'test',
+        },
       });
+      expect(wrapper.text()).toContain('test');
+    });
 
-      it('should render the slot when the slot has content', () => {
-        const wrapper = shallowMount(KButton, {
-          propsData: {
-            text: 'test',
-          },
-          slots: {
-            default: '<span>slot</span>',
-          },
-        });
-        expect(wrapper.text()).toContain('slot');
-        expect(wrapper.text()).toContain('test');
+    it('should render the slot when the slot has content', () => {
+      const wrapper = shallowMount(KButton, {
+        propsData: {
+          text: 'test',
+        },
+        slots: {
+          default: '<span>slot</span>',
+        },
       });
+      expect(wrapper.text()).toContain('slot');
+      expect(wrapper.text()).toContain('test');
     });
+  });
 
-    describe('event handling', () => {
-      it('should emit a click event when clicked', () => {
-        const wrapper = shallowMount(KButton, {
-          propsData: {
-            text: 'test',
-          },
-        });
-        wrapper.trigger('click');
-        expect(wrapper.emitted().click).toBeTruthy();
+  describeUnit('event handling', () => {
+    it('should emit a click event when clicked', () => {
+      const wrapper = shallowMount(KButton, {
+        propsData: {
+          text: 'test',
+        },
       });
+      wrapper.trigger('click');
+      expect(wrapper.emitted().click).toBeTruthy();
     });
-  } else {
-    describe('KButton Visual Tests', () => {
-      beforeAll(async () => {
-        await page.goto('http://localhost:4000/testing-playground', { waitUntil: 'networkidle2' });
-      });
+  });
 
-      async function renderComponent(component, props) {
-        await page.evaluate(
-          ({ component, props }) => {
-            window.postMessage(
-              {
-                type: 'RENDER_COMPONENT',
-                component: component,
-                props: props,
-              },
-              '*'
-            );
-          },
-          { component, props }
-        );
-        await page.waitForSelector('#testing-playground');
-
-        const isComponentRendered = await page.evaluate(() => {
-          const testing_playground = document.querySelector('#testing-playground');
-          return testing_playground && testing_playground.children.length > 0;
-        });
-
-        if (!isComponentRendered) {
-          // eslint-disable-next-line no-console
-          console.error('Component did not render in the testing playground');
-        }
-      }
-
-      it('renders correctly with default props', async () => {
-        await renderComponent('KButton', { text: 'Test Button' });
-        await percySnapshot(page, 'KButton - Default');
-      });
+  describeVisual('KButton Visual Tests', () => {
+    it('renders correctly with default props', async () => {
```

Comment:
Wooo, loved it, it looks super great and easy to use :tada: :star_struck:

Reply:
Thanks a lot :smile: 
I will work on the suggestions provided and update this PR by tomorrow.